### PR TITLE
Initial version

### PR DESCRIPTION
**Pull request by EdmondIguazio**
All the offline datasets tree are under /v3io/bigdata/demos-data/iguazio/. 
I should be created on cluster creation.

### CODE REVIEW

File: getting-started-tutorial/01-mlrun-basics.ipynb [hunk 1]
```diff
@@ -399,24 +399,52 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "## Set this environment variable in case of dark site\n",
+    "# os.environ['SAMPLE_DATA_SOURCE_URL_PREFIX'] = '/v3io/bigdata/demos-data/iguazio/'"
+   ]
+  },
```
Comment (gilad-shaham) [inline getting-started-tutorial/01-mlrun-basics.ipynb:418]: I think we should give instructions in markdown, we don't need environment variables if the user is going to change the value within the notebook.
Also, in the text, if we give a reference to a folder, we should have it in /v3io/projects, not bigdata

File: getting-started-tutorial/01-mlrun-basics.ipynb [hunk 2]
```diff
@@ -399,24 +399,52 @@
...
+    "url_prefix = os.environ.get('SAMPLE_DATA_SOURCE_URL_PREFIX', 'https://s3.wasabisys.com/iguazio/')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 31,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "source_url = f'{url_prefix}data/iris/iris.data.raw.csv'"
```
Comment (gilad-shaham) [inline getting-started-tutorial/01-mlrun-basics.ipynb:435]: We need to consider what happens if we don't have / at the end of url_prefix. So we need the code the work the same if we have a trailing '/' or if we don't have.
So I think this is better source_url = f"{url_prefix.strip('/')}/data/iris/iris.data.raw.csv"

File: getting-started-tutorial/01-mlrun-basics.ipynb [hunk 3]
```diff
@@ -133,15 +133,15 @@
...
+   "execution_count": 22,
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Project name: getting-started-tutorial-iguazio\n",
-      "Artifacts path: /v3io/projects/{{run.project}}/artifacts\n"
+      "Project name: getting-started-tutorial-edmond\n",
```
Comment (gilad-shaham) [inline getting-started-tutorial/01-mlrun-basics.ipynb:143]: Probably better to run the code with user iguazio to keep the text output the same

File: getting-started-tutorial/04-pipeline.ipynb [hunk 4]
```diff
@@ -131,15 +131,15 @@
...
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Project path: /User/demos/getting-started-tutorial/conf\n",
-      "Project name: getting-started-tutorial-iguazio\n"
+      "Project path: /User/demos-dark/getting-started-tutorial/conf\n",
+      "Project name: getting-started-tutorial-edmond\n"
```
Comment (gilad-shaham) [inline getting-started-tutorial/04-pipeline.ipynb:142]: Better to run in the original folder + use the user iguazio so the output is the same as before

File: getting-started-tutorial/04-pipeline.ipynb [hunk 5]
```diff
@@ -354,11 +354,13 @@
     "from kfp import dsl\n",
     "import mlrun\n",
     "from mlrun.platforms import auto_mount\n",
-    "\n",
+    "import os\n",
     "\n",
     "funcs = {}\n",
     "DATASET = 'cleaned_data'\n",
     "LABELS = \"label\"\n",
+    "# os.environ['SAMPLE_DATA_SOURCE_URL_PREFIX'] = '/v3io/bigdata/demos-data/iguazio/'\n",
```
Comment (gilad-shaham) [inline getting-started-tutorial/04-pipeline.ipynb:362]: Comment probably not necessary

File: getting-started-tutorial/04-pipeline.ipynb [hunk 6]
```diff
@@ -372,7 +374,8 @@
     "                \"capabilities of the Iguazio Data Science Platform.\\n\"\n",
     "                \"The tutorial uses the Iris flower data set.\"\n",
     ")\n",
-    "def kfpipeline(source_url='https://s3.wasabisys.com/iguazio/data/iris/iris_dataset.csv'):\n",
+    "\n",
+    "def kfpipeline(source_url=f'{url_prefix}data/iris/iris_dataset.csv'):\n",
```
Comment (gilad-shaham) [inline getting-started-tutorial/04-pipeline.ipynb:378]: same as before for the join

**Review by gilad-shaham — CHANGES_REQUESTED** [review]

File: getting-started-tutorial/01-mlrun-basics.ipynb (hunk 1 above)
Comment (EdmondIguazio) [reply getting-started-tutorial/01-mlrun-basics.ipynb:418]: Instructions are now in markdown and folder changed to projects

File: getting-started-tutorial/01-mlrun-basics.ipynb (hunk 2 above)
Comment (EdmondIguazio) [reply getting-started-tutorial/01-mlrun-basics.ipynb:435]: Modifed, used `rstrip` instead

File: getting-started-tutorial/01-mlrun-basics.ipynb (hunk 3 above)
Comment (EdmondIguazio) [reply getting-started-tutorial/01-mlrun-basics.ipynb:143]: OK

File: getting-started-tutorial/04-pipeline.ipynb (hunk 5 above)
Comment (EdmondIguazio) [reply getting-started-tutorial/04-pipeline.ipynb:362]: Removed

File: getting-started-tutorial/04-pipeline.ipynb (hunk 6 above)
Comment (EdmondIguazio) [reply getting-started-tutorial/04-pipeline.ipynb:378]: Modified

**Comment by EdmondIguazio** [comment]
@gilad-shaham 
Resolved the issues, please approve

**Pull request merged**